Packs/Gamma/Integrations/Gamma/Gamma.yml

```diff
@@ -0,0 +1,178 @@
+category: Network Security
+commonfields:
+  id: Gamma
+  version: -1
+configuration:
+- additionalinfo: URL of the Gamma API to query
+  defaultvalue: https://app.gamma.ai
+  display: Server URL (e.g. https://example.net)
+  name: url
+  required: true
+  type: 0
+- display: Gamma API Key
+  hidden: false
+  name: api_key
+  required: true
+  type: 4
+- additionalinfo: Enter the violation ID to begin fetching from. If empty, the API
+    will default to your first violation
+  display: Starting Violation
+  hidden: false
+  name: first_fetch_violation
+  required: false
+  type: 0
+- additionalinfo: Enter a number of results for each fetch to return. The number must
+    be between 1 and 100.
+  defaultvalue: '10'
+  display: Results per fetch
+  hidden: false
+  name: max_results
+  required: false
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  type: 13
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+  required: false
+description: Query and update violations in Gamma
+display: Gamma
+name: Gamma
+script:
+  commands:
+  - arguments:
+    - default: false
+      defaultValue: '1'
+      description: Violation ID to begin pulling from. Defaults to the earliest existing
+        violation for your account.
+      isArray: false
+      name: minimum_violation
+      required: true
+      secret: false
+    - default: false
+      defaultValue: '10'
+      description: 'Upper limit for retrieving violations (max: 100)'
+      isArray: false
+      name: limit
+      required: true
+      secret: false
+    deprecated: false
+    description: Get a list of violations
+    execution: false
+    name: gamma-get-violation-list
+    outputs:
+      - contextPath: response.violation_id
```

Comment:
According to the code, the basic context key is `GammaViolation`, so this context path (and all the rest) can't be correct.
Please run the command and see what are the real paths.

Reply:
you are right, fixed it now.